src/readers/morphologyASC.cpp

```diff
@@ -40,6 +40,18 @@ bool is_neurite_type(Token id) {
             id == Token::CELLBODY);
 }
 
+std::string as_token_text(const std::string& text) {
+    std::string upp_text = text;
```

Comment:
I'm open to suggestions here as I'm not C++ expert. It does:
- makes text upper case
- removes any whitespace

Reply:
Nope, any. it's for making `Cell Body` into `CellBody`.

Reply:
Thanks.